syntax = "proto2";
import "ignition/msgs/header.proto";
package smartmouse.msgs;

message ServerControl {
    optional ignition.msgs.Header header = 1;
    optional bool pause = 2;
    optional bool static = 3;
    optional uint32 step = 4;
    optional bool quit = 7;
    optional bool reset_time = 8;
    optional bool reset_robot = 9;
    optional double reset_row = 10;
    optional double reset_col = 11;
    optional double reset_yaw = 12;
    optional string author = 13;
    optional bool toggle_play_pause = 14;
}

